addon/-private/system/relationships/ext.js

```diff
@@ -171,7 +171,7 @@ export const RelationshipsClassMethodsMixin = Ember.Mixin.create({
     });
    ```
 
-    Calling `App.Post.typeForRelationship('comments')` will return `App.Comment`.
+    Calling `postRecord.constructor.typeForRelationship('comments')` will return `Comment`.
```

Comment:
Hmm, also it looks like `typeForRelationship` needs the `store` to be passed as 2nd argument...